
package com.barryku.cloud.sforce.partner;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.4.2
 * 2011-09-11T16:54:21.738-07:00
 * Generated source version: 2.4.2
 */

@WebFault(name = "LoginFault", targetNamespace = "urn:fault.partner.soap.sforce.com")
public class LoginFault_Exception extends Exception {
    
    private com.barryku.cloud.sforce.partner.LoginFault loginFault;

    public LoginFault_Exception() {
        super();
    }
    
    public LoginFault_Exception(String message) {
        super(message);
    }
    
    public LoginFault_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public LoginFault_Exception(String message, com.barryku.cloud.sforce.partner.LoginFault loginFault) {
        super(message);
        this.loginFault = loginFault;
    }

    public LoginFault_Exception(String message, com.barryku.cloud.sforce.partner.LoginFault loginFault, Throwable cause) {
        super(message, cause);
        this.loginFault = loginFault;
    }

    public com.barryku.cloud.sforce.partner.LoginFault getFaultInfo() {
        return this.loginFault;
    }
}
